### PR TITLE
update geizhals doc

### PR DESCRIPTION
**Pull request by JulianKahnert**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15990

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.geizhals.markdown [hunk 1]
```diff
@@ -23,16 +22,16 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
 sensor:
   - platform: geizhals
     name: qc35
-    product_id: 1453021
+    product_id: 1696985
 ```
 
 Configuration variables:
```
Comment (DubhAd) [inline source/_components/sensor.geizhals.markdown:28]: As you're updating the config block anyway, now would be an ideal time to update this to use the new format as detailed in the documentation standards ;)

Comment (JulianKahnert) [reply source/_components/sensor.geizhals.markdown:28]: What do you mean by that?

Comment (DubhAd) [reply source/_components/sensor.geizhals.markdown:28]: https://developers.home-assistant.io/docs/en/documentation_standards.html#component-and-platform-pages

**Comment by frenck** [comment]
@JulianKahnert That is not required. It is always nice for people to update the documentation as a whole. But just adding the thing that is needed is OK as well.

✅ Approved.

**Review by frenck — APPROVED** [review]

**Pull request merged**